dependencies.yaml

```diff
@@ -40,5 +41,12 @@ dependencies:
       version: "2.24.0"
       lang: java
       scope: test
+  org.eclipse.californium:
```

Comment:
hmm it seems califronium is `Eclipse Californium is a Java implementation of RFC7252 - Constrained Application Protocol for IoT Cloud services` is library which focuses on specific goals and need. Wouldn't it be better to use generic cryptography library which supports both tls and dtls, like https://bouncycastle.org/ ?

Reply:
As far as I know, the scandium subproject contains a complete implementation of DTLS and the code quality is certainly very good. Nonetheless, I wish I had been aware of bouncy castle's DTLS support. I will investigate in the background.